commands/fncmd.go

```diff
@@ -54,6 +55,8 @@ func GetFnCommand(name string) *cobra.Command {
 	sink.Long = fndocs.SinkShort + "\n" + fndocs.SinkLong
 	sink.Example = fndocs.SinkExamples
 
-	functions.AddCommand(run, source, sink)
+	export := cmdexport.ExportCommand()
```

Comment:
Can we add `export` to `kpt fn` command after the `export` supports different orchestrators?
Otherwise any kpt being released with have incomplete support for `kpt export`.